src/lib/accountBalances.js

```diff
@@ -22,6 +23,20 @@ AccountBalances.prototype.applyDebits = function * () { yield this._applyDebits(
 AccountBalances.prototype.applyCredits = function * () { yield this._applyCredits(this._credits) }
 AccountBalances.prototype.revertDebits = function * () { yield this._applyCredits(this._debits) }
 
+function sum (numbers) {
+  const numStrings = _.map(numbers, String)
+  return _.reduce(_.rest(numStrings), (result, num) => {
```

Comment:
Oh, I guess you would have to do the BigNumber cast in the function then, so nevermind.